src/pages/MoviePage/AddToFavouriteButton/index.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState, useCallback } from 'react';
+import { IoIosHeartEmpty, IoIosHeart } from 'react-icons/io';
+
+import { Container } from './styles';
+
+export default function AddToFavouriteButton() {
+	const [favourite, setFavourite] = useState(false);
+
+	const handleAddToFavourites = useCallback(() => {
```

Comment:
Great use of `useCallback`, because you have sure that `favourite` has changed before update it again. Another approach could be:

```ts
const handleAddToFavourites = () => {
   setFavourite(prev => !prev);
}
```